cpu/stm32_common/periph/i2c_2.c

```diff
@@ -174,6 +177,39 @@ static void _i2c_init(I2C_TypeDef *i2c, uint32_t clk, uint32_t ccr)
     i2c->CR1 |= I2C_CR1_PE;
 }
 
+static void _re_init(i2c_t dev)
```

Comment:
Do you mind refactoring the i2c_init() to use this so we don't add all the code size?

Reply:
refactored. I don't have the hardware to test at the moment though.